docs/rules/no-restricted-imports.md

```diff
@@ -0,0 +1,51 @@
+# Disallow Node imports (no-restricted-imports)
+
+Disallowing usage of specific node imports can be useful if you want to control the available methods, a developer can
+use, to implement a feature.
+
+This way you can block usage of the `fs` import if you want disallow file system access.
+Blocking the `os` module can be useful if you don't want to allow any operating system specific code.
+
+## Rule Details
+
+This rule allows you to specify imports that you don't want to use in your application.
+
+### Options
+
+The syntax to specify restricted modules looks like this:
+
+```json
+"no-restricted-imports": [2, <...importNames>]
```

Comment:
This is invalid JSON add will cause a parse error.  Maybe replace with "import 1", " import2" to make it valid?